static/app/views/starfish/components/tableCells/spanDescriptionCell.tsx

```diff
@@ -41,18 +43,42 @@ export function SpanDescriptionCell({
     />
   );
 
-  return ModuleName.DB ? (
-    <DescriptionWrapper>
-      <WiderHovercard
-        position="right"
-        body={<FullQueryDescription group={group} shortDescription={description} />}
-      >
-        {descriptionLink}
-      </WiderHovercard>
-    </DescriptionWrapper>
-  ) : (
-    descriptionLink
-  );
+  if (moduleName === ModuleName.DB) {
+    return (
+      <DescriptionWrapper>
+        <WiderHovercard
+          position="right"
+          body={<FullSpanDescription group={group} shortDescription={description} />}
+        >
+          {descriptionLink}
+        </WiderHovercard>
+      </DescriptionWrapper>
+    );
+  }
+
+  if (moduleName === ModuleName.HTTP) {
+    return (
+      <DescriptionWrapper>
+        <WiderHovercard
+          position="right"
+          body={
+            <Fragment>
+              {t('Example')}
+              <FullSpanDescription
```

Comment:
l: Can we add some padding between `Example` and the description?

Reply:
👍